Core/Items/PoTGlobalItem.Data.cs

```diff
@@ -48,9 +56,9 @@ public override GlobalItem Clone(Item from, Item to)
 	public Influence Influence { get; set; }
 
 	/// <summary>
-	///		The formatted, post-rolled name containing formatting for rarities.
+	///		The localization IDs for the prefix/suffix. Defaults to -1, -1.
 	/// </summary>
-	public string SpecialName { get; set; } = string.Empty;
+	public NameAffixes NameAffix { get; set; } = new();
```

Comment:
This avoids the POTO crash. Loss of function is less important.
```suggestion
	public NameAffixes NameAffix { get; set; } = new();
	
	[Obsolete($"Use {nameof(NameAffix)}", error: true)]
	public string SpecialName { get => string.Empty; set { } }
```

Other than that, approved for when this and the magic constant issues are solved.